modules/viz/samples/viz_sample_03.py

```diff
@@ -0,0 +1,60 @@
+#exec(open(r'G:\Lib\opencv_contrib\modules\viz\samples\viz_sample_03.py').read())
+import os
+os.add_dll_directory(r'G:\Lib\install\opencv\x64\vc15\bin')
+os.add_dll_directory(r'G:\Lib\install\vtk\bin')
+os.add_dll_directory(r'G:\Lib\install\ceres-solver\bin')
+os.add_dll_directory(r'G:\Lib\install\glog\bin')
+os.add_dll_directory(r'F:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.2\bin')
+os.add_dll_directory(r'F:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.2')
```

Comment:
It is for Python 3.8+ on Windows only.
There is no good solution for that from Python team.

There is some attempt to handle OpenCV paths through `<build_dir>/python_loader`.

Anyway such hacks should be removed from the source code.